src/HuC1.md

```diff
@@ -1,50 +1,51 @@
 # HuC1
 
-HuC1 is an MBC used by some Game Boy games which besides doing the usual
-MBC stuff, also provides IR comms. A lot of sources on the internet said
-that HuC1 was \"similar to MBC1\", but they didn\'t provide any detail.
-I took a look, and it turns out that HuC1 differs from MBC1 quite a lot.
+HuC1 is an MBC used by some Game Boy games which, besides doing the
+usual MBC stuff, also provides infrared communication. A lot of sources
+on the internet said that HuC1 was “similar to MBC1”, but they didn’t
+provide any detail. I took a look, and it turns out that HuC1 differs
```

Comment:
```suggestion
provide any detail. The HuC1 differs
```

Since we're here already, those kind of comments/remarks should be removed

Reply:
This page is basically copied from a jrra.zone blog post (hopefully with permission): http://jrra.zone/blog/huc1.html

I wasn’t editing it for style, I don’t know whether the license of the blog post would even allow that.  There’s no clear copyright notice/license on the original blog post, so Berne Convention automatic copyright/all rights reserved applies.

Reply:
All content under Pan Docs has been licensed to the public domain, so that should be fine.

Reply:
I tightened up the wording in a few places. here.